platform/index.md

```diff
@@ -1,18 +1,70 @@
+
+In order to allow ServiceControl for gathering information, you need to configure your solution properly:
+
+* [enable auditing](nservicebus/operations/auditing.md) to collect data on individual messages;
+* [install plugins on your endpoints](servicecontrol/plugins.md) to monitor their health, sagas and use custom checks.
+
+By default ServiceControl stores information for 30 days and initiates purging process every minute, but you can easily customize those values using [configuration](servicecontrol/creating-config-file.md) or [command-line arguments](servicecontrol/how-purge-expired-data.md).
+
+The standard approach is to have one ServiceControl installation on a single host. ServicePulse and ServiceInsight can be installed on other machines. You need to configure ServicePulse and ServiceInsight to connect with ServiceControl. You can also make ServiceControl highly available, to avoid having a single point of failure in your monitoring system.
+
+# ServiceInsight - invaluable developer tool
+
+ServiceInsight is a WPF application with features tailored to developers needs. It allows for advanced debugging, tracking the flow of the individual message in the system, observing sagas and more. 
+
+Thanks to the rich visual content, such as flow diagrams, it is much easier to quickly spot anomalies and incorrect behaviour. At the same time, it gives access to more detailed information, such as message headers and all message metadata. For more information, including sample screens refer to [documentation](serviceinsight/getting-started-overview.md).
+
+
+# ServicePulse - production monitoring
+
+ServicePulse is a web application dedicated mainly to administrators. It gives you a clear, near real-time, high-level overview of the system. 
+
+There you will get notified when the endpoint is down or when a message fails. You can also specify your own custom checks and get alerts. The interface allows you to perform the common operations for failure recovery, such as restarting failed messages. You can also [subscribe to publicly exposed events](servicepulse/event-types.md), in order to display and handle them in a custom way.
+
+
+# ServiceMatrix - boost your productivity
+
+ServiceMatrix is a VisualStudio extension, that allows for creating messaging system with NServiceBus using visual tools. With a few simple clicks you can generate a fully functional solution and even deploy it. You can also easily extend the generated code.
+
+ServiceMatrix is perfect for Proof of Concept, prototyping and discussing project design while coding kind of work. It also is invaluable when the team is new to messaging concepts, as it is much easier to understand how system works and how information flows through it, having a visual representation of the system that corresponds to the actual code.
```

Comment:
I like the first part. - `ServiceMatrix is perfect for Proof of Concept, prototyping and discussing project design`
Not sure about what follows next. 
Some of it sounds more like marketing than on the documentation. 

How about something like this:
`ServiceMatrix is perfect for proof of concept, prototyping and discussing project design with a team of developers. Having a working prototype with visualization helps to clarify the design better in terms of how the system works. Combined with the run time visualization from ServiceInsight, the team can see how information and messages flow through the various endpoints using message flow and sequence diagrams.`